docs/config.ld

```diff
@@ -456,6 +456,25 @@ add_custom_tag {
     },
 }
 
+-- Define the supermodule class.
+-- This tag should be used at the module level. All properties from the
+-- supermodule will be recursively added to the module by our ldoc template.
+-- @supermodule supermodule
+add_custom_tag {
+    name = "supermodule",
+    hiden = true,
+    auto_subtags = false
+}
+
+-- Mark the item ad hiden.
+-- This tag should be used to hide items from the documentation.
+-- @hiden
+add_custom_tag {
+    name = "hiden",
+    hiden = true,
```

Comment:
This should be "hide" or "hidden". "hiden" doesn't exist.

Reply:
Oops... And I did this typo everywhere!